pkg/internal/codegen/parse/util.go

```diff
@@ -326,3 +340,21 @@ func checkType(props *v1beta1.JSONSchemaProps, s string, enums *[]v1beta1.JSON)
 		*enums = append(*enums, v1beta1.JSON{Raw: []byte(`"` + s + `"`)})
 	}
 }
+
+// Scale subresource requires specpath, statuspath, selectorpath key values, represents for JSONPath of
+// SpecReplicasPath, StatusReplicasPath, LabelSelectorPath separately. e.g.
+// +kubebuilder:subresource:scale:specpath=.spec.replica,statuspath=.status.replica,selectorpath=
+func parseJSONPath(t *types.Type) map[string]string {
```

Comment:
This function can encounter errors during parsing, so it should return error.

Reply:
Done